src/js/ico.js.rst

```diff
@@ -0,0 +1,263 @@
+ICO Handling
+------------
+Utilities for generating and handling ICO data.
+
+
+Create an ICO from PNG Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Follows `Kevin Mårtensson`_'s `to-ico`_. See the `ICO File Format`_
+specification on Wikipedia for reference.
+
+.. _Kevin Mårtensson: https://github.com/kevva
+.. _to-ico: https://github.com/kevva/to-ico
+.. _ICO File Format: https://en.wikipedia.org/wiki/ICO_%28file_format%29#Outline
+
+The ICO file header block is six bytes, or three 16-bit integers written in
```

Comment:
This is cool!

Reply:
These are the echos of a C struct from 25 years ago...